code/game/objects/structures/window.dm

```diff
@@ -397,12 +397,25 @@ var/global/wcCommon = pick(list("#379963", "#0d8395", "#58b5c3", "#49e46e", "#8f
 	return TRUE
 
 /obj/structure/window/AltClick(mob/user)
-	if(user.incapacitated())
-		to_chat(user, "<span class='warning'>You can't do that right now!</span>")
-		return
-	if(!Adjacent(user))

```

Comment:
Please re-add this important check. Without it you can rotate the windows from anywhere on the screen using your short cut.